huaweicloud-sdk-python-modelarts-dataset/modelarts/test/test_manifest_classification.py

```diff
@@ -15,18 +15,21 @@
 import os
 import sys
 
-import manifest
+from modelarts import manifest
 
 
 def validate(data_set):
   assert data_set.get_size() > 18
   data_objects = data_set.get_sample_list()
   for data_object in data_objects:
-    assert str(data_object.get_source()).__contains__("s3://obs-ma/test/flowers/datafiles/")
+    assert str(data_object.get_source()).__contains__("s3://obs-ma/test/classification/datafiles/")
```

Comment:
why not use `in` method instead of `__contains__()`

Reply:
What benefit for using in method instead of __contains__()?